scilpy/tracking/tools.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import numpy as np
+
+DETERMINISTIC = 'det'
```

Comment:
Since those are defined here, you should import the definitions in your other script instead of repeating it.